back/engines/commercial/analysis/app/controllers/analysis/web_api/v1/taggings_controller.rb

```diff
@@ -39,13 +39,11 @@ def destroy
         def bulk_create
           @filters = filters(params[:filters])
           @inputs = InputsFinder.new(@analysis, @filters).execute
+          taggings = []
           @inputs.each do |input|
-            begin
-              Tagging.create!(tag_id: params[:tag_id], input_id: input.id)
-            rescue ActiveRecord::RecordInvalid => e
-              puts e.message
-            end
+            taggings.push({ tag_id: params[:tag_id], input_id: input.id })
```

Comment:
Nothing wrong with the code, just making you aware there is also a `map` in Ruby for this

Reply:
Ah, good to know 👍